2025-04-19 14:27:01,367 INFO    MainThread:2628231 [wandb_setup.py:_flush():76] Configure stats pid to 2628231
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_setup.py:_flush():76] Loading settings from /home/md5121/.config/wandb/settings
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_setup.py:_flush():76] Loading settings from /scratch/md5121/okhatisathi/tacotron2/wandb/settings
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_init.py:_log_setup():461] Logging user logs to /scratch/md5121/okhatisathi/tacotron2/wandb/run-20250419_142701-p94125xa/logs/debug.log
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_init.py:_log_setup():462] Logging internal logs to /scratch/md5121/okhatisathi/tacotron2/wandb/run-20250419_142701-p94125xa/logs/debug-internal.log
2025-04-19 14:27:01,368 INFO    MainThread:2628231 [wandb_init.py:_jupyter_setup():411] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x1554cab817f0>
2025-04-19 14:27:01,369 INFO    MainThread:2628231 [wandb_init.py:init():495] calling init triggers
2025-04-19 14:27:01,369 INFO    MainThread:2628231 [wandb_init.py:init():499] wandb.init called with sweep_config: {}
config: {'epochs': 5, 'iters_per_checkpoint': 100, 'seed': 1234, 'dynamic_loss_scaling': True, 'fp16_run': False, 'distributed_run': False, 'dist_backend': 'nccl', 'dist_url': 'tcp://localhost:54321', 'cudnn_enabled': True, 'cudnn_benchmark': False, 'ignore_layers': ['embedding.weight'], 'load_mel_from_disk': False, 'training_files': './filelists/train_list.txt', 'validation_files': './filelists/val_list.txt', 'text_cleaners': ['transliteration_cleaners'], 'max_wav_value': 32768.0, 'sampling_rate': 22050, 'filter_length': 1024, 'hop_length': 256, 'win_length': 1024, 'n_mel_channels': 80, 'mel_fmin': 0.0, 'mel_fmax': 8000.0, 'n_symbols': 148, 'symbols_embedding_dim': 512, 'encoder_kernel_size': 5, 'encoder_n_convolutions': 3, 'encoder_embedding_dim': 512, 'n_frames_per_step': 1, 'decoder_rnn_dim': 1024, 'prenet_dim': 256, 'max_decoder_steps': 10000, 'gate_threshold': 0.5, 'p_attention_dropout': 0.1, 'p_decoder_dropout': 0.1, 'attention_rnn_dim': 1024, 'attention_dim': 128, 'attention_location_n_filters': 32, 'attention_location_kernel_size': 31, 'postnet_embedding_dim': 512, 'postnet_kernel_size': 5, 'postnet_n_convolutions': 5, 'use_saved_learning_rate': False, 'learning_rate': 0.001, 'weight_decay': 1e-06, 'grad_clip_thresh': 1.0, 'batch_size': 3, 'mask_padding': True}
2025-04-19 14:27:01,369 INFO    MainThread:2628231 [wandb_init.py:init():548] starting backend
2025-04-19 14:27:01,369 INFO    MainThread:2628231 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-19 14:27:01,380 INFO    MainThread:2628231 [backend.py:ensure_launched():217] starting backend process...
2025-04-19 14:27:01,389 INFO    MainThread:2628231 [backend.py:ensure_launched():222] started backend process with pid: 2628288
2025-04-19 14:27:01,391 INFO    MainThread:2628231 [wandb_init.py:init():558] backend started and connected
2025-04-19 14:27:01,405 INFO    MainThread:2628231 [wandb_run.py:_label_probe_notebook():1084] probe notebook
2025-04-19 14:27:01,406 INFO    MainThread:2628231 [wandb_run.py:_label_probe_notebook():1094] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-19 14:27:01,406 INFO    MainThread:2628231 [wandb_init.py:init():634] updated telemetry
2025-04-19 14:27:01,454 INFO    MainThread:2628231 [wandb_init.py:init():666] communicating run to backend with 30 second timeout
2025-04-19 14:27:03,003 INFO    MainThread:2628231 [wandb_run.py:_on_init():1919] communicating current version
2025-04-19 14:27:03,081 INFO    MainThread:2628231 [wandb_run.py:_on_init():1923] got version response upgrade_message: "wandb version 0.19.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-04-19 14:27:03,081 INFO    MainThread:2628231 [wandb_init.py:init():700] starting run threads in backend
2025-04-19 14:27:06,199 INFO    MainThread:2628231 [wandb_run.py:_console_start():1893] atexit reg
2025-04-19 14:27:06,201 INFO    MainThread:2628231 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.WRAP
2025-04-19 14:27:06,203 INFO    MainThread:2628231 [wandb_run.py:_redirect():1803] Wrapping output streams.
2025-04-19 14:27:06,210 INFO    MainThread:2628231 [wandb_run.py:_redirect():1827] Redirects installed.
2025-04-19 14:27:06,210 INFO    MainThread:2628231 [wandb_init.py:init():732] run started, returning control to user process
2025-04-19 14:54:46,466 INFO    MainThread:2628231 [wandb_init.py:_pause_backend():374] pausing backend
2025-04-19 14:54:46,470 INFO    MainThread:2628231 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2025-04-19 15:15:20,882 INFO    MainThread:2628231 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2025-04-19 15:15:20,885 INFO    MainThread:2628231 [wandb_run.py:_restore():1834] restore
2025-04-19 15:15:21,217 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 733
}

2025-04-19 15:15:21,529 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 733
}

2025-04-19 15:15:22,000 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 63414
}

2025-04-19 15:15:22,101 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 63414
}

2025-04-19 15:15:22,202 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 63414
}

2025-04-19 15:15:22,303 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 63414
}

2025-04-19 15:15:22,404 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 733
  total_bytes: 63414
}

2025-04-19 15:15:22,505 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:22,606 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:22,707 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:22,808 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:22,909 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:23,010 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:23,111 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

2025-04-19 15:15:23,212 INFO    MainThread:2628231 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 63414
  total_bytes: 63414
}

